coordinator/src/main/scala/filodb.coordinator/client/QueryCommands.scala

```diff
@@ -60,15 +60,16 @@ object QueryCommands {
     * This class provides general query processing parameters
     * @param spreadFunc a function that returns chronologically ordered spread changes for the filter
     */
-  final case class QueryOptions(spreadProvider: SpreadProvider = StaticSpreadProvider(),
+     // to do see if can make spreadProvider as Options[Int] issue with QueryEngine
+  final case class QueryOptions(spread: Option[Int]= None,
```

Comment:
spacing around `=`

Reply:
Actually do we really want to replace the spreadProvider?   I think it is better to keep this API no for flexibility?

Reply:
@velvia Vish suggested that Spreadprovider should not be a part of QueryOptions. I changed it to spread as input and created SpreadProvider in QueryActor

Reply:
We could, but the param would then have to be `Option[SpreadProvider]` instead since it is an override which by default wont be provided. I was originally in favor of simplicity since this is an override really. But I am ok with keeping it an `Option[SpreadProvider]` too - if the changes are straightforward.

Reply:
@velvia  Is Option[Int] fine or should I change it to Option[SpreadProvider]?

Reply:
The main argument for `Option[SpreadProvider]` is flexibility in case a power user wants to use Akka API to provide their own.  However, whatever SpreadProvider is included basically would have to be in the FiloDB codebase when deployed.  It's for future extensibility.

Reply:
> The main argument for `Option[SpreadProvider]` is flexibility in case a power user wants to use Akka API to provide their own. However, whatever SpreadProvider is included basically would have to be in the FiloDB codebase when deployed. It's for future extensibility.

Changed to Option[SpreadProvider]